tests/Tools/ProtoLoader.php

```diff
@@ -37,7 +37,10 @@ public static function loadDescriptorBytes(string $protoPath): string
         // Set up required file locations and create tmp output file for protoc invocation.
         // Assumes test are executed from within the repo root directory.
         $cwd = getcwd();
-        $protoc = "{$cwd}/tools/protoc";
+        $protoc = "protoc";
+        if (getenv("USE_TOOLS_PROTOC")) {
```

Comment:
if `export USE_TOOLS_PROTOC=`, then this will be false but the bash conditional will be true. to ensure there's no discrepancy, I'd check for the string `true` in both cases, or the string `1`, but be specific.

Reply:
Done! Switched it to compare to `'true'`.

Reply:
Another good option is `filter_var($var, FILTER_VALIDATE_BOOLEAN)`. This will give you an actual boolean to work with regardless of casing, etc.